sentry-ruby/spec/sentry/interfaces/request_interface_spec.rb

```diff
@@ -4,15 +4,11 @@
 
 RSpec.describe Sentry::RequestInterface do
   let(:env) { Rack::MockRequest.env_for("/test") }
+  let(:send_default_pii) { false }
+  let(:rack_env_whitelist) { Sentry::Configuration::RACK_ENV_WHITELIST_DEFAULT }
 
   subject do
-    described_class.build(env: env)
-  end
-
-  before do
-    Sentry.init do |config|
```

Comment:
It also makes tests cleaner, as we don't need to initialize the entire SDK for testing an interface class.